<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoScanner</class>
 <widget class="QWidget" name="AutoScanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QComboBox" name="combo_baudrate_scanner"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_com_scanner">
     <property name="text">
      <string>COM &amp; BAUD</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="combo_comport_scanner">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_message_scanner">
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="line_message_scanner"/>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="btn_apply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_status_scanner">
     <property name="text">
      <string>Status Scaner</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="btn_open_scanner">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
